tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -8507,6 +8507,7 @@ declare namespace ts {
     function getNameOfDeclaration(declaration: Declaration | Expression | undefined): DeclarationName | undefined;
     function getDecorators(node: HasDecorators): readonly Decorator[] | undefined;
     function getModifiers(node: HasModifiers): readonly Modifier[] | undefined;
+    function getJSDocCommentsAndTags(hostNode: Node, noCache?: boolean): readonly (JSDoc | JSDocTag)[];
```

Comment:
Do we have `noCache` anywhere else in the public code? Is this useful externally?

Reply:
I must not have been paying much attention, didn't mean to expose that parameter externally, I don't think it should be used

Reply:
I've moved it back to utilities.ts, and avoided exposing `noCache` externally, which unfortunately required either an extra function, or an overload and disabling a rule.